### PR TITLE
Feature search focus

### PR DESCRIPTION
**Pull request by meggles**
Custom focus indicator for search page

### CODE REVIEW

File: src/components/Algolia/shared/RefinementFilter/RefinementFilter.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const RefinementFilterLabelTheme = {
     }
 
     &:focus-within {
-      border: 1px dashed ${color.eclipse};
+      outline: 2px dotted ${color.eclipse};
```
Comment (ttornese) [inline src/components/Algolia/shared/RefinementFilter/RefinementFilter.js:43]: What are your thoughts on making this focus outline treatment a mixin? Maybe with a color as an argument? We could also possibly import that mixin into espresso/jarvis from mise.

Comment (meggles) [reply src/components/Algolia/shared/RefinementFilter/RefinementFilter.js:43]: I think that's a good idea! I will add that in.

File: src/components/Algolia/shared/RefinementFilter/RefinementFilter.js [hunk 2]
```diff
@@ -40,7 +40,8 @@ const RefinementFilterLabelTheme = {
     }
 
     &:focus-within {
-      border: 1px dashed ${color.eclipse};
+      ${mixins.focusIndicator(color.eclipse)};
+      outline-offset: 0;
```
Comment (meggles) [inline src/components/Algolia/shared/RefinementFilter/RefinementFilter.js:44]: Want to point out that I think `outline-offset` will usually be set to `3px`, but a couple of the algolia components look a little better without an offset so I have overridden it.

**Review by ttornese — APPROVED** [review]
🔥

**Review by mirdathebirda — APPROVED** [review]

File: src/styles/mixins.js [hunk 3]
```diff
@@ -78,4 +78,11 @@ export default {
       }
     `;
   },
+
+  focusIndicator(color) {
```
Comment (jaslakson) [inline src/styles/mixins.js:82]: would be be good to have a default here - whatever the most common situation would be. If we had to change it in the future we might be able to change fewer placements if we leverage a default.

Comment (meggles) [reply src/styles/mixins.js:82]: good call – I will add a default.

**Review by jaslakson — APPROVED** [review]

**Pull request merged**